### PR TITLE
Fix tests

### PR DESCRIPTION
**Pull request by nbierdeman**
### Purpose

This PR fixes the failing tests as noted in the comments below.

### CODE REVIEW

**Comment by nbierdeman** [comment]
@wsbrunson @westeezy @gregjopa, this fixed the tests for me locally, so I wanted to see what you think. Here's a screenshot of the failing tests before they were fixed:

Screen Shot 2021-12-17 at 11 07 14 AM
.

**Review by westeezy — APPROVED** [review]
lgtm though we might consider pinning flow versions if flow-typed step fails (that relies on pinned flow version)

**Comment by nbierdeman** [comment]
@westeezy, `flow-typed` is passing for me now, so I'm thinking we leave it as is until something changes. That alright?

**Review by wsbrunson — APPROVED** [review]
niiiiice

File: package.json [hunk 1]
```diff
@@ -54,6 +54,6 @@
   "dependencies": {
     "@paypal/sdk-constants": "^1.0.11",
     "belter": "^1.0.45",
-    "jsx-pragmatic": "^2"
+    "jsx-pragmatic": "2.0.20"
```
[the remark this answers is not in the document]
Comment (nbierdeman) [reply package.json:57]: Yeah, sure thing! Please let me know if you can think of a better way of testing `flow` versions, other than guessing and checking. 😬

**Comment by nbierdeman** [comment]
@gregjopa, I paired with @westeezy on finding a decent compromise on pinning an exact version of flow without having to upgrade grumbler-scripts or make changes to jsx-pragmatic for now. What do you think of where we landed?

**Review by gregjopa — APPROVED** [review]
Awesome work @nbierdeman and @westeezy! 💯

**Pull request merged**